packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -200,6 +209,12 @@ export const VideoGallery = (props: VideoGalleryProps): JSX.Element => {
   const containerHeight = _useContainerHeight(containerRef);
   const isNarrow = containerWidth ? isNarrowWidth(containerWidth) : false;
 
+  /* @conditional-compile-remove(pinned-participants) */
+  const [pinnedParticipantsState, _] = useState<string[]>([]);
+  /* @conditional-compile-remove(pinned-participants) */
+  // Use pinnedParticipants from props but if it is not defined use the maintained state of pinned participants
+  const pinnedParticipants = props.pinnedParticipants ? props.pinnedParticipants : pinnedParticipantsState;
```

Comment:
Another way is
`const pinnedParticipants = props.pinnedParticipants ?? pinnedParticipantsState;`